package.json

```diff
@@ -42,12 +41,12 @@
 		"prettier": "^1.15.3"
 	},
 	"scripts": {
-		"test": "cross-env PORT=8081 jest",
+		"test": "SET PORT=8081 jest",
```

Comment:
This will break scripts for us on non-windows machines, please revert to old package.json.